assets/locales/debugger.properties

```diff
@@ -253,7 +253,7 @@ editor.conditionalPanel.placeholder=This breakpoint will pause when the expressi
 
 # LOCALIZATION NOTE(editor.jumpToMappedLocation): Context menu item
 # for navigating to a source mapped location
-editor.jumpToMappedLocation=Jump to %s location
+editor.jumpToMappedLocation=Jump to %S location
```

Comment:
In this case you need a new string ID, the variable changes, dashboard will throw errors.